src/horus_api.c

```diff
@@ -68,16 +68,44 @@ int8_t uw_horus_rtty[] = {
   0,0,1,0,0,1,0,1,1,0
 };
 
-/* Unique word for Horus Binary */
+/* Unique word for Horus Legacy Binary */
 
-int8_t uw_horus_binary[] = {
+int8_t uw_horus_legacy_binary[] = {
     0,0,1,0,0,1,0,0,
```

Comment:
Lets discuss names of the protocols .... legacy/new doesn't leave room for expansion should we develop a 3rd or 4th version.  Maybe sequential letters horus_binary_a/horus_binary_b or numbers horus_binary_v1/v2?

Reply:
I figure the newer modes are still in flux for now, hence Legacy for the known-working mode. The newer mode names can be changed. 

I've left all the defines as HORUS_MODE_BINARY for the legacy mode for now.

Reply:
... so lets put some thought into it and sort the names out in this PR.